js/src/api/transport/ws/ws.js

```diff
@@ -18,6 +18,7 @@ import { keccak_256 } from 'js-sha3'; // eslint-disable-line camelcase
 
 import { Logging } from '../../subscriptions';
 import JsonRpcBase from '../jsonRpcBase';
+import TransportError from '../error';
```

Comment:
Any reason why this is only introduced to the WS transport and not the HTTP transport (as used by dapps) as well?